utilities/rpp-unittests/HIP_NEW/Tensor_hip_pkd3.cpp

```diff
@@ -978,6 +993,107 @@ int main(int argc, char **argv)
 
         break;
     }
+    case 82:
+    {
+        test_case_name = "ricap";
```

Comment:
The HIP test suite for ricap should ideally look exactly like the host test suite except for 2-3 lines where we have hipMemcpy() for ROIs and hipDeviceSynchronize(). So one might need to change.